### DISCUSSION TITLE
Skip download java db when generate image SBOM only

### DISCUSSION THREAD

**Discussion by ngocnn97** [aquasecurity/trivy#5941] [Q&A]
### Question

I just need to generate SBOM image but Trivy requires updating the java database. I tried --skip-java-db-update but it didn't run in the first time.
I also don't need to download java database because I just need to generate SBOM from the image.
So . How can I skip the step of downloading java db, when generating SBOM image.

### Target

SBOM

### Scanner

None

### Output Format

CycloneDX

### Mode

Standalone

### Operating System

Linux

### Version

```bash
Version: 0.46.0
```

**Comment by knqyf263** [comment]
The Java DB is necessary to detect installed libraries.

**Reply by ngocnn97** [reply]
We have already scanned the java libraries in the filesystem scan step, is it necessary to scan further in the image scan step?

**Reply by knqyf263** [reply]
If you want to skip Java analysis, it is another feature request. You can track this issue. Please reopen this discussion if your request is something else.
https://github.com/aquasecurity/trivy/issues/3987
